packages/react-core/src/components/Select/Select.tsx

```diff
@@ -428,6 +429,13 @@ export class Select extends React.Component<SelectProps & OUIAProps, SelectState
                   (valueToCheck as SelectOptionObject).toString &&
                   (valueToCheck as SelectOptionObject).compareTo;
 
+                // View more option should be returned as not a match
+                if (loadingVariant !== undefined && loadingVariant !== 'spinner') {
+                  if (loadingVariant?.text === valueToCheck && loadingVariant.text === valueToCheck) {
+                    return true;
+                  }
+                }
```

Comment:
```suggestion
                if (loadingVariant?.text === valueToCheck) {
                  return true;
                }
```

Reply:
That gave me an error when I coded it that way because loading variant might not have `.text` if if is spinner.  So I had to add the check.

Reply:
So I assume it could just be `if (loadingVariant !== 'spinner' && loadingVariant?.text === valueToCheck) {`, yes?